Week3/homework/src/addRemainder.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function addRemainder(connection, errorHandler, req, res) {
+  const username = req.params.username;
+  const password = req.params.password;
+  const listDescription = req.params.listDescription;
+  const remainder = req.params.remainder;
+  const queryString1 = 'SELECT user_id FROM users WHERE user_name = ? AND user_password = ?;';
+  connection.query(queryString1, [username, password], (err, rows) => {
+    errorHandler(err, res);
+    const queryString2 = 'UPDATE lists SET remainder = ? WHERE user_id = ? AND description = ?';
```

Comment:
Seems like you're relying here on `description` in WHERE condition. 
I advise against such a thing, because of imagine the situation:
- you had list description
- the description has changed because of, let's say user edited it
- your query wouldn't be able to find the list to update anymore. 

And also, have a look - you use `user_id` already to identify the user, and this is a reliable approach. 
Please use primary key of  `lists` there

Reply:
URI is changed and connected to user_id as PK.